timemachine/fe/functional.py

```diff
@@ -13,38 +16,61 @@ def _make_selection_mask(compute_du_dx=False, compute_du_dp=False, compute_du_dl
 
 
 def wrap_impl(impl, pack=lambda x: x):
```

Comment:
Before this is ready for review, I need to resolve an issue with how I'm handling `pack(params)`

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/707/commits/09c0406635c6ceab0a740dbee938ad7441f253fa